index.html

```diff
@@ -6462,6 +6636,28 @@ <h2 id="13Decoders.Errors">Error handling</h2>
 "chunk">iCCP</span></a>) could lead to buffer overruns.
 Implementors of deflate decompressors should guard against this
 possibility.</p>
+
+<p>
+  APNG is designed to allow incremental display of frames
+  before the entire image has been read.
```

Comment:
Is this PR intended to be the APNG spec integrated as-is? Cause that's an unfortunate way to word it, frames are positioned after the default image, it reads as if the opposite is the case.

"... designed to allow incremental display of frames before the entire sequence has been read." would make more sense.

Reply:
It isn't integrated fully as-is, but I retained the original wording unless there was good reason to change it, and put the various parts in the right places in the PNG spec.

Good point about "entire image" being misleading; PNG defines the concept of [datastream](https://w3c.github.io/PNG-spec/#5DataRep) so that would be good here: "entire datastream".